app/api/v0/clusters/route.ts

```diff
@@ -10,15 +10,23 @@ export const GET = withAuth(async ({ client, user }) => {
 
   const { data, error } = await client
     .from("clusters")
-    .select("id:index, nickname, description, hardware, cycle_type, proof_type")
+    .select(
+      `
+      id:index, nickname, description, hardware, cycle_type, proof_type,
+      cluster_configuration:cluster_configurations(
```

Comment:
renaming for clarity to API consumers here

Reply:
This pattern will be extended to #142